server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -144,84 +143,43 @@ export class MetadataExtractionProcessor {
   async extractExifInfo(job: Job<IExifExtractionProcessor>) {
     try {
       const { asset, fileName }: { asset: AssetEntity; fileName: string } = job.data;
-      const exifData = await exifr.parse(asset.originalPath, {
-        tiff: true,
-        ifd0: true as any,
-        ifd1: true,
-        exif: true,
-        gps: true,
-        interop: true,
-        xmp: true,
-        icc: true,
-        iptc: true,
-        jfif: true,
-        ihdr: true,
-      });
+      const exifData = await exiftool.read(asset.originalPath);
 
       if (!exifData) {
```

Comment:
This line will not do anything because you will always have `exifData` if a file doesn't have exif data an error will be thrown. So you might have to do a catch directly in the exif data read method 

```typescript
const exifData = await exiftool.read(asset.originalPath).catch(e => console.log("No exif"));
```

And then resolve the scenario when no EXIF data is available.